app/src/main/java/org/fossasia/openevent/app/core/sponsor/list/SponsorsPresenter.java

```diff
@@ -97,7 +97,7 @@ public void loadSponsors(boolean forceReload) {
         return sponsors;
     }
 
-    private void deleteSponsor(Long sponsorId) {
+    public void deleteSponsor(Long sponsorId) {
```

Comment:
Should be package protected and with annotation `@VisibleForTesting`

Reply:
It gives `error: getSelectedSponsors() is not public in SponsorsPresenter; cannot be accessed from outside package` when no modifier is used, and `.. is protected...` when protected is used.

Reply:
If you are testing its functionality in some other way, then don't explicitly test it

Reply:
`deleteSponsor()` is indirectly tested when we test `deleteSelectedSponsors()`. So I guess I should remove its test? But for `getSelectedSponsors()`, the above error persists because of different package. Should I make it public again?

Reply:
You should add test in the same package as presenter